### PR TITLE
[Docenhancement][Sideloadingnote]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added note for sideloading app.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/get-started/get-started-use-app-studio.md](https://github.com/MicrosoftDocs/msteams-docs/blob/80859b5e7f040c43fa2435eb286a787a08e1aba5/msteams-platform/includes/get-started/get-started-use-app-studio.md) | 91 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/21f6cdd9-a6b8-4f23-b7ac-8fcde5241bad) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/get-started/get-started-use-app-studio.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d3cb47a19670aed8b050ff4ac9af28b0cdbd057d/msteams-platform/includes/get-started/get-started-use-app-studio.md) | 91 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0e6d5cc-a8a7-443e-8f7d-811db09d1c19) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — DISMISSED** [review]
Approve

File: msteams-platform/includes/get-started/get-started-use-app-studio.md [hunk 1]
```diff
@@ -162,6 +162,9 @@ After entering the details of your app, complete the following steps to register
 1. To upload your app to Teams, select the **Install** button under **Test and Distribute**:
 
     <img  width="450px" alt="Adding a messaging extension dialog" src="~/assets/images/get-started/InstallingHelloWorld.png"/>
+    
+    > [!NOTE]
+    > If you are unable to sideload the app, verify whether you have [prepared your development environment](#prepare-your-development-environment).
```
Comment (Wajeed-msft) [inline msteams-platform/includes/get-started/get-started-use-app-studio.md:167]:
```suggestion
    > If you are unable to sideload the app, verify whether you have [enabled custom app uploading](#prepare-your-development-environment).
```

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/includes/get-started/get-started-use-app-studio.md](https://github.com/MicrosoftDocs/msteams-docs/blob/caea5ddb7313f48d72a3ba78a62e3395df6818e5/msteams-platform/includes/get-started/get-started-use-app-studio.md) | 91 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa6b24dd-da47-4628-aaee-8a93206456c6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**